docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '2'
+services:
+  web:
+    image: 'reload/drupal-apache-fpm'
+    ports:
+      - 80
+    volumes_from:
+      - webroot
+    environment:
+      VIRTUAL_HOST: 'kbhlydavis.docker'
+
+  fpm:
+    image: 'reload/drupal-php7-fpm:5.6'
+    environment:
+      PHP_SENDMAIL_PATH: /usr/local/bin/mhsendmail --smtp-addr="mailhog:1025"
+    volumes_from:
+      - webroot
+    ports:
+      - 9000
+
+  db:
+    image: 'mysql:5.6'
+    ports:
+      - 3306
+    volumes:
+      - ./kbhlydavis.sql:/docker-entrypoint-initdb.d/kbhlydavis.sql
+    environment:
+      MYSQL_DATABASE: 'db'
+      MYSQL_USER: 'db'
+      MYSQL_PASSWORD: 'db'
+      MYSQL_ROOT_PASSWORD: 'db'
```

Comment:
Hvorfor har du de her environments med når det alligevel er hardcoded i settingsfilen?
Jeg forstår godt hvorfor de er hardcoded, da environment ting ikke virker længere i docker service 2 :)

Reply:
De skal sættes for at MySQL docker imaget kan sættes op.
Den skal vide, hvad navnet på databasen skal være samt oprette en MySQL bruger med et password.